code/game/objects/items/plushes.dm

```diff
@@ -682,6 +677,18 @@
 	user.dust(just_ash = FALSE, drop_items = TRUE)
 	return MANUAL_SUICIDE
 
+/obj/item/toy/plush/moth/suicide_desc(lives_taken)
+	if(lives_taken <= 0)
```

Comment:
You added the component for this, now you need to remove this functionality or integrate it

Reply:
what do u mean

Reply:
im trying to *not* remove functionality, and it would look weird to have it display the count twice